src/auth/index.ts

```diff
@@ -1014,6 +1014,11 @@ export namespace auth {
       passwordRequired?: boolean;
     };
 
+    /**
+     * Whether the anonymous provider is enabled.
+     */
+    anonymousSignInEnabled?: boolean;
```

Comment:
Is this really optional? Since `Tenant` is mainly an output type I'd think this can always be present. In fact it is always present in the `Tenant` implementation class.